host/sgx/windows/enter.asm

```diff
@@ -40,10 +40,11 @@ ARG1            EQU [rbp-24]
 ARG2            EQU [rbp-32]
 ARG3            EQU [rbp-40]
 ARG4            EQU [rbp-48]
-ENCLAVE          EQU [rbp-56]
+ENCLAVE         EQU [rbp-56]
 ARG1OUT         EQU [rbp-64]
 ARG2OUT         EQU [rbp-72]
 STACKPTR        EQU [rbp-80]
+MXCSR           EQU [rbp-88]
```

Comment:
Since we are adding space to store the mxcsr value, shouldn't the PARAM space also increase?

Reply:
We allot more for PARAM_SPACE (i.e. 128) as stack needs to 16-byte aligned for various reasons - IA64 calling convention and efficiency as this permits simple data alignment especially for SSE registers.

Reply:
In linux we do an explicit alignment after return from ecall. Is that not needed for windows?
https://github.com/Microsoft/openenclave/blob/master/host/sgx/linux/enter.S#L98

Reply:
@anakrish - Yes, the explicit alignment after return from ecall is done on Windows too. On Linux, PARAMS_SPACE is forced to be at 16-byte alignment boundary by padding an extra byte.